src/third_party/cudasift/CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+cmake_minimum_required(VERSION 2.8)
+
+project(cudasift)
+set(cudaSift_VERSION_MAJOR 2)
+set(cudaSift_VERSION_MINOR 0)
+set(cudaSift_VERSION_PATCH 0)
+
+set(CPACK_PACKAGE_VERSION_MAJOR "${cudaSift_VERSION_MAJOR}")
+set(CPACK_PACKAGE_VERSION_MINOR "${cudaSift_VERSION_MINOR}")
+set(CPACK_PACKAGE_VERSION_PATCH "${cudaSift_VERSION_PATCH}")
+set(CPACK_GENERATOR "ZIP")
+include(CPack)
+
+if (WIN32)
+  set(EXTRA_CXX_FLAGS "/DVERBOSE /D_CRT_SECURE_NO_WARNINGS ")
+  list(APPEND CUDA_NVCC_FLAGS "-arch=sm_35;--compiler-options;-O2;-DVERBOSE") 
```

Comment:
That's strange. I reused the same arch variable that was originally in the CMakeLists (sm_35), but I have a Pascal GPU (1070) on CUDA 8, which I think should need compute_61+sm_61 but it compiles without issues for me. I assumed find(CUDA) was automatically appending the right arch flag.. I might be wrong though.